### PR TITLE
blog pages

### PR DESCRIPTION
**Pull request by Southclaws**
A simple Markdown (MDX) blog system for sharing open.mp updates with the community.

### CODE REVIEW

File: posts/turfs-formerly-gangzones-module.mdx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+export const meta = {
+  date: '2019-10-19T04:20:00',
+  author: 'J0sh'
+};
+
+# Turfs (formerly gangzones) module
```
Comment (jtylr) [inline posts/turfs-formerly-gangzones-module.mdx:6]: Probably more of a question for Josh - but should this be just "Turf"? plural being "Some turf"

See also: Lego/some lego/a piece of lego vs "Legos"

Comment (Southclaws) [reply posts/turfs-formerly-gangzones-module.mdx:6]: Probably should be Turf, yeah.

Also, this whole code-review process is going to make putting out content very easy and transparent! We can catch typos and stuff like this early then once it's live, copy it out to facebook/burgershot/twitter/etc

**Comment by Southclaws** [comment]
Still left to do: proper styling that's readable.

However, I may just do this in another PR because the whole site needs a new strategy for CSS - I want to move it to a component model for easier development and easier transition from what the site is now to a fully fledged landing page for the project.

**Comment by Southclaws** [comment]
Another reason is I would like to just get the blog out soon so we can do a better job of keeping the community up to date!

**Pull request merged**